.gitignore

```diff
@@ -1,4 +1,10 @@
 .hg*
+
+# Celery/Gravity state files
+
+celerybeat-schedule.*
+database/gravity
```

Comment:
Should we just replace the over 20 `database/something` entries with just `database/*` ?

Reply:
Maybe just add (exclude) a README with a comment (like the current info.txt) and ignore the rest?

Reply:
IMO just ignore the whole thing.  If someone needs to change the info or add more trakced files, that's always still possible.

Reply:
I pushed a commit doing this; hope that's fine @davebx